lib/ansible/modules/identity/ipa/ipa_host.py

```diff
@@ -183,6 +189,8 @@ def get_host_dict(description=None, force=None, ip_address=None, ns_host_locatio
         data['usercertificate'] = [{"__base64__": item} for item in user_certificate]
     if mac_address is not None:
         data['macaddress'] = mac_address
+    if userpassword is not None:
```

Comment:
To keep with the convention of the other parameters, I think it would make sense to rename this to be `user_password` (with the underscore).

Reply:
The naming follows the naming as it uses the FreeIPA API. It is ok to adapt the name to the other parameters. Shall I? Do you do?

Reply:
Yes, I think the underscore will keep the consistency with the other parameters. For example, the module attribute `mac_address` correlates to the FreeIPA API option `macaddress`.

This would be the same; the module attribute `user_password` correlates to the FreeIPA API option `userpassword`.